packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -63,11 +68,16 @@ class InputDateTimePicker extends React.Component {
 		}
 
 		checkSupportedDateFormat(props.dateFormat);
+		const selectedDate =
+			props.useTime === false
```

Comment:
if default is `false` no need to test if it's explicitly false, just use the truthyness `props.useTime ? ... : ...`